src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -484,6 +484,35 @@ $__Command $HARNESS_RUNNER apple just-run %5c
   -v
 CLRTestExitCode=$?
 
+# Exit code of xharness is zero when tests finished successfully
```

Comment:
Why do we need special-case `maccatalyst` now or to be precise why `just-run` is not permitted?
How did it work before running `maccatalyst` tests with Mono?

Reply:
If I remember correctly, `run` accepts path to the app which it installs and runs (for all platforms). For maccatalyst it technically just calles `open [app]`.

The `just-run` command accepts app name and just opens the app on the device without installation. maccatalyst needs to have the app bundle path because technically we don't install anything anywhere. So it could technically get the path instead of the app bundle name but I guess it just didn't make sense because of this.

Reply:
Thanks for the explanation, I am good with the changes.